sections/sections-app/src/java/org/sakaiproject/tool/section/jsf/backingbean/OptionsBean.java

```diff
@@ -82,7 +82,7 @@ public void init() {
 			management = INTERNAL;
 			// Once manual has been chosen, prevent instructors to switch to externally managed.
 			boolean restrictExternallyManagedSections = ServerConfigurationService.getBoolean(RESTRICT_EXTERNALLY_MANAGED_SECTIONS_PROP, false);
-			managementToggleEnabled = ! (restrictExternallyManagedSections && !isSuperUser());
+			managementToggleEnabled = ((!ExternalIntegrationConfig.MANUAL_MANDATORY.equals(config)) && ! (restrictExternallyManagedSections && !isSuperUser()));
```

Comment:
@jesusmmp if the Section is internally managed why would external integration have anything to do at this point?

Reply:
I don't really understand the question... If the MANUAL_MANDATORY is selected the TA should not be able to change the selector.

Sakai 22 with MANUAL_MANDATORY
![image](https://github.com/user-attachments/assets/84e56152-572c-43ea-acf5-680bcc6fc004)

Reply:
Sakai 25 with MANUAL_MANDATORY (with this patch)

![image](https://github.com/user-attachments/assets/b8bcd6d1-c82f-43f7-95c9-206f93e42d27)